src/main/java/entities/ObservableGame.java

```diff
@@ -24,36 +24,4 @@ public interface ObservableGame {
      * Completely clear the list of observers, such that it is now empty.
      */
     void deleteObservers();
-
-    /**
-     * Get the Player whose turn it is currently.
-     * @return The Player in this Game who needs to put down a card (i.e. the player whose current turn it is).
-     */
-    Player getCurrentTurn();
-
-    /**
-     * Get the Card that had been played last.
-     * @return A Card object which the current Player places their card on top of (i.e. the topmost card in the pile).
-     */
-    Card getCurrentCard();
-
-    /**
-     * Get the Card that was drawn last.
-     * @return A Card object which the current Player has drawn from the deck.
-     */
-    Card getDrawnCard();
-
-    /**
-     * Get all the Players in this Game.
-     * @return An ArrayList of Players in this game.
-     */
```

Comment:
ITs good that the interface has been rid of all these methods,, interface was pretty heavy there.